src/main/java/games/strategy/util/Version.java

```diff
@@ -175,12 +180,12 @@ public String toStringFull(final String separator) {
   }
 
   public String toStringFull(final String separator, final boolean noMicro) {
```

Comment:
I'm not sure if this method is 100% still needed, could you please check on that @simon33-2 . IIRC, it was added to have a way to display the full 5 digit version number.

Regardless, this can be simplified, this version of `toStringFull` can be made private, then inlined with the caller, and `noMicro` is always false.

Reply:
> IIRC, it was added to have a way to display the full 5 digit version number.

I'm not 100% on this, but I think `getExactVersion()` is used for that purpose (at least that's what's used by `GameSelectorPanel` to display the engine version on the main menu).

Reply:
I've done that, but that is completely outside the scope of what I was attempting here.